.firebaserc

```diff
@@ -1,5 +1,6 @@
 {
   "projects": {
-    "default": "rocketry-minerva"
+    "default": "rocketry-minerva",
+    "development": "rocketry-minerva-dev"
```

Comment:
Should production really be default? Shouldn't it be harder to deploy to production than development?

Reply:
Good point. Swapped.